package.json

```diff
@@ -60,16 +62,19 @@
     "jsonpath": "^1.0.0",
     "lodash": "^4.17.5",
     "virtual-alexa": "^0.6.9",
-    "virtual-device-sdk": "^1.4.10"
+    "virtual-device-sdk": "^1.4.10",
+    "virtual-google-assistant": "0.2.1-0"
```

Comment:
This version will change once the changes on virtual google assistant are approved and the official 0.2.1 version is released. (This is the beta one)